### PR TITLE
feat: Add support for mnemonic word

### PR DESCRIPTION
**Pull request by Jack-Works**
👀 DONT MERGE
Need security audit by @yisiliu 

# Create new key by mnemonic word

image

image

# Attach new account to another account (which means they share the same crypto key)
image

# Recover key by mnemonic word
image

### CODE REVIEW

**Comment by Jack-Works** [comment]
@Tedko You can make demo by code on this branch

**Comment by Tedko** [comment]
Ok. Will take a look at it.

File: src/network/gun/version.2/hash.ts [hunk 1]
```diff
@@ -24,11 +25,17 @@ export const hashPostSalt = memoizePromise(async function(postSalt: string) {
     return hash.substring(0, N)
 })
 
-export const hashCryptoKey = memoizePromise(async function(key: CryptoKey) {
+/**
+ * @param key - The key need to be hashed
+ * @param stableHash - Set this to true if you're writing new code.
+ * Unstable hash may cause problem but we cannot just switch to stable hash
+ * because it may breaks current data.
```
Comment (Artoria2e5) [inline src/network/gun/version.2/hash.ts]: Aren't those both JSON anyway?

**Comment by Tedko** [comment]
Works well on my side, prepared a short video : https://mega.nz/#!upRzXKaK!BIG7QvYJV-mkiebNGnCZPB6i5lwOpGhlmoxVR57jHQs

**Pull request merged**